Kiss.Bff/Groepen/GroepenProxyConfig.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Net.Http.Headers;
+using Yarp.ReverseProxy.Transforms;
+
+namespace Kiss.Bff.Groepen
+{
+    public static class GroepenExtensions
```

Comment:
afdelingen en groepen horen heel erg bij elkaar. 1 organisatorische eenheden configuratie (baseurl, proxy, etc) lijkt me toepasselijker. is dat overwogen?

Reply:
Ja, en ik twijfel nog steeds :). Uiteindelijk ben ik hiervoor gegaan omdat objecten en objecttypes ook losse apis zijn, en contactmomenten en klanten ook. Maar ik sta ervoor open om er 1 ding van te maken

Reply:
besproken